src/components/Interactive/MyFilter.vue

```diff
@@ -0,0 +1,186 @@
+                       chosenLeftReleaseDateVote: leftYearValue,
+                       chosenRightReleaseDateVote: rightYearValue,
+                       selectedValue})"
+                   v-model="rightRatingValue" min="0" max="10" id="upperRating">
+          </span>
+        <p v-if="leftRatingValue === rightRatingValue">Only with {{ leftRatingValue }}</p>
+        <p v-else>from {{ leftRatingValue }} to {{ rightRatingValue }}</p>
+      </b-col>
+    </div>
+    <div style="margin-top: 70px;">
+      <h5>By Year:</h5>
+      <b-col sm="12">
+          <span class="multi-range">
+            <input type="range"
+                   @change="changeLeftYearRating(leftYearValue), getFilter({
+                       withGenres:selectedGenres,
+                       chosenLeftRatingVote: leftRatingValue,
+                       chosenRightRatingVote: rightRatingValue,
+                       chosenLeftReleaseDateVote: leftYearValue,
+                       chosenRightReleaseDateVote: rightYearValue,
+                       selectedValue})"
+                   v-model="leftYearValue" min="1895" max="2022" id="lowerYear">
+            <input type="range"
+                   @change="changeRightYearRating(rightYearValue), getFilter({
+                       withGenres:selectedGenres,
+                       chosenLeftRatingVote: leftRatingValue,
+                       chosenRightRatingVote: rightRatingValue,
+                       chosenLeftReleaseDateVote: leftYearValue,
+                       chosenRightReleaseDateVote: rightYearValue,
+                       selectedValue})"
+                   v-model="rightYearValue" min="1895" max="2022" id="upperYear">
+          </span>
+        <p v-if="leftYearValue === rightYearValue">Only with {{ leftYearValue }}</p>
+        <p v-else>from {{ leftYearValue }} to {{ rightYearValue }}</p>
+      </b-col>
+    </div>
+
+    <div class="mt-5" style="width: 250px; margin: 0 auto;">
+      <b-form-select v-model="selected" :options="options" @input="changeSelectValue(selected), getFilter({
+                       withGenres:selectedGenres,
+                       chosenLeftRatingVote: leftRatingValue,
+                       chosenRightRatingVote: rightRatingValue,
+                       chosenLeftReleaseDateVote: leftYearValue,
+                       chosenRightReleaseDateVote: rightYearValue,
+                       selectedValue})">
+        <template #first>
+          <b-form-select-option :value="null" disabled>Please select an sort option</b-form-select-option>
+        </template>
+      </b-form-select>
+    </div>
+  </div>
+</template>
+
+<script>
+import {mapActions, mapMutations, mapState} from "vuex";
+import MyTags from "@/components/Lists/MyTags";
+
+export default {
+  name: "MyFilter",
+  components: {MyTags},
+  data: () => ({
+    leftRatingValue: '0',
+    rightRatingValue: '10',
+    leftYearValue: '1895',
+    rightYearValue: '2022',
+    selected: null,
+    options: [
```

Comment:
If this is a constant that never changes, like a configuration variable here, use CAPS_CASE for it.